### ISSUE TITLE
Remove unused dependencies from mirrord-layer

### ISSUE THREAD

**Issue by abhirag** [metalbear-co/mirrord#220]
`mirrord-layer` has the following unused dependencies -- 
```
mirrord-layer v2.5.0 (/home/parallels/Documents/mirrord_build_time/mirrord/mirrord-layer)
└─── dependencies
     ├─── "clap"
     ├─── "multi-map"
     ├─── "nanoid"
     ├─── "nix"
     └─── "queues"
```
- [ ] Verify the analysis using [`cargo-udeps`](https://github.com/est31/cargo-udeps)
- [ ] Remove the dependencies from [`Cargo.toml`](https://github.com/metalbear-co/mirrord/blob/main/mirrord-layer/Cargo.toml)
- [ ] Make sure that the build succeeds

**Comment by neimsaci** [comment]
Hi I am beginner and it looks like this issue is for me! :D

**Comment by eyalb181** [comment]
@neimsaci All yours! I've assigned you the issue, if you need any help feel free to ask here or on [Discord](https://discord.gg/pSKEdmNZcK). Thanks!

**Comment by aviramha** [comment]
oops. We forgot to close this! Fixed in #243
